src/kudu/consensus/raft_consensus.cc

```diff
@@ -519,7 +519,10 @@ Status RaftConsensus::Start(const ConsensusBootstrapInfo& info,
 
   if (IsSingleVoterConfig() && FLAGS_enable_leader_failure_detection) {
     LOG_WITH_PREFIX(INFO) << "Only one voter in the Raft config. Triggering election immediately";
-    RETURN_NOT_OK(StartElection(NORMAL_ELECTION, { INITIAL_SINGLE_NODE_ELECTION }));
+    RETURN_NOT_OK(
+        StartElection(NORMAL_ELECTION, {INITIAL_SINGLE_NODE_ELECTION,
```

Comment:
curious, did we add these enum values? ELECTION_TIMEOUT_EXPIRED/INITIAL_SINGLE_NODE_ELECTION